test/verify/main.cpp

```diff
@@ -79,7 +79,6 @@ int main(int argc, const char* argv[])
          "test_resize_dyn",
          "test_instancenorm_large_3d<migraphx::shape::float_type>",
          "test_instancenorm_large_3d<migraphx::shape::half_type>",
-         "test_multinomial<migraphx::shape::bf16_type>",
```

Comment:
Why enable on cpu?

Reply:
This was for disabling on CPU. Removed it since we're commenting it out for now.